src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/touchstoneRepository/GetCoverageDataForResponsibilityTests.kt

```diff
@@ -37,26 +37,29 @@ class GetCoverageDataForResponsibilityTests : TouchstoneRepositoryTests()
                     scenarioId)
 
             assertThat(result.toList()).containsExactlyElementsOf(listOf(
-                    LongCoverageRow(scenarioId, "First", "AF", GAVISupportLevel.WITHOUT, ActivityType.ROUTINE,
-                            "AAA", "AAA-Name", 2001, 2.toDecimal(), 4.toDecimal(), null, null, null),
+                    GenderedLongCoverageRow(scenarioId, "First", "AF", GAVISupportLevel.WITHOUT, ActivityType.ROUTINE,
```

Comment:
So `touchstoneVersionId` here is `touchstone-1`, as declared in the base class - is it clear that we should be returning gender for touchstone ids without an explicit date in the id? 

In any case could we move the declaration of `touchstoneVersionId` into the test itself, so that it's easy to read this test and see that it's returning the appropriate data given the `touchstoneVersionId`? Or could create a shared variable called `2019touchstoneVersionId` to make it clear. Would also be good to have a test with an id like `2020-somethingorother`.

Reply:
I'm not sure we really have clearly defined requirements for non-standard touchstone names, but I guess the new behaviour should be the default. Anyway, I'll change this to use a 2019 touchstone Id, and also 2020.

Reply:
Cool, just had a quick check and there are no touchstones in the db that don't start with a year :+1: